samples/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -148,6 +149,19 @@ static int assign_handles(struct bt_gatt_dm *dm)
 	return err;
 }
 
+void hidc_write_cb(struct bt_gatt_hids_c *hidc,
+		      struct bt_gatt_hids_c_rep_info *rep,
+		      u8_t err)
+{
+	if (err) {
+		LOG_WRN("Failed to write report via HIDS: %d", err);
```

Comment:
` via HIDS` is not needed as logged module name will show where problem occured

Reply:
Removed